packages/terra-alert/package.json

```diff
@@ -26,19 +26,15 @@
   "peerDependencies": {
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
-    "terra-base": "^3.7.0",
-    "terra-button": "^2.8.0",
-    "terra-doc-template": "^1.3.0",
-    "terra-icon": "^2.7.0",
-    "terra-responsive-element": "^2.7.0"
+    "terra-base": "^3.12.0"
   },
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
-    "terra-base": "^3.12.0",
     "terra-button": "^2.13.0",
+    "terra-doc-template": "^1.6.0",
     "terra-icon": "^2.13.0",
-    "terra-responsive-element": "^3.0.0",
+    "terra-responsive-element": "^2.12.0",
```

Comment:
Should this be using the `^3.0.0`?

Reply:
yes. the latest pr release caused a merge conflict in each file I had changes so I must have missed this one.